### PR TITLE
doc(atomic): dispatch event for Atomic production releases

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/KIT-858

### CODE REVIEW

File: scripts/notify-docs/published-headless-atomic.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const {Octokit} = require('@octokit/rest');
```
Comment (ThibodeauJF) [inline scripts/notify-docs/published-headless-atomic.js:1]: Updated the file to support atomic & headless, I didn't change the structure very much

File: scripts/notify-docs/published-headless-atomic.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  try {
+    await notify('published_headless_to_npm', {headless_version: headlessPackageJson.version});
+    console.log('headless notification sent');
+  } catch(e) {
+    console.log('headless notification failed');
+    logError(e);
+  }
+
+  try {
+    await notify('published_atomic_to_npm', {atomic_version: atomicPackageJson.version});
```
Comment (ThibodeauJF) [inline scripts/notify-docs/published-headless-atomic.js:30]: @dlutzCoveo I guess this event (`published_atomic_to_npm`) has to be added to the `doc_jekyll-public-site` repo, that's how you would be able to access the docs

Comment (dlutzCoveo) [reply scripts/notify-docs/published-headless-atomic.js:30]: Perfect! I'll add that to the action, thanks!

File: scripts/notify-docs/published-headless-atomic.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+const owner = 'coveo';
+const repo = 'doc_jekyll-public-site';
+
+async function notify(event_type, client_payload) {
+  return github.repos.createDispatchEvent({owner, repo, event_type, client_payload});
+}
+
+function logError(e) {
+  const {status, message, request} = e;
+    console.log(status, message, request);
```
Comment (olamothe) [inline scripts/notify-docs/published-headless-atomic.js:17]:
```suggestion
    console.error(status, message, request);
```

File: scripts/notify-docs/published-headless-atomic.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+async function main() {
```
Comment (olamothe) [inline scripts/notify-docs/published-headless-atomic.js:20]: This works and is simple enough, so I'm fine with it.

I think however the "cleanest" would have been to accept which package to notify as script arguments.

```
node notify-doc.js -package headless@1.2345 -package atomic@9.876 - package quantic@3.1416 -package ...
```

**Review by olamothe — APPROVED** [review]

**Review by nbuissoncoveo — APPROVED** [review]

**Review by btaillon — APPROVED** [review]

**Review by dlutzCoveo — APPROVED** [review]

**Pull request merged**